OpenSim/Sandbox/MatlabScripts/opensimTimeSeriesTableToMatlab.m

```diff
@@ -33,15 +44,15 @@
        new_col_label = col_label(temp(end-1)+1:temp(end)-1);
```

Comment:
What will happen here if the table contains coordinate values? I think then data will contain only one column, called `data.value`. What do you think about converting the slashes into underscores, instead?